h2/src/main/org/h2/util/JdbcUtils.java

```diff
@@ -415,6 +421,25 @@ public static Object deserialize(byte[] data, JavaObjectSerializer javaObjectSer
         }
     }
 
+    /**
+     * De-serialize the byte array to a UUID object.
```

Comment:
best to document the reason for this, something like

"to avoid a security hole with a malicously crafted serialised object"